Jasmine/Common/Models/GameInfo.swift

```diff
@@ -0,0 +1,6 @@
+struct GameInfo {
+    let levelName: String
+    let gameType: GameType
+    let gameMode: GameMode
+    let gameViewModel: BaseViewModelProtocol
```

Comment:
@xdrawks Showing high score may be very distracting and I didn't account for this in models :X